src/binder/expression_binder.cpp

```diff
@@ -351,8 +351,7 @@ shared_ptr<Expression> ExpressionBinder::bindNodeLabelFunction(const Expression&
     nodeLabels.resize(maxNodeTableID + 1);
     for (auto i = 0; i < nodeLabels.size(); ++i) {
         if (catalogContent->containNodeTable(i)) {
-            auto tableSchema = catalogContent->getNodeTableSchema(i);
-            nodeLabels[i] = Literal(tableSchema->tableName);
+            nodeLabels[i] = Literal(catalogContent->getTableSchema(i)->tableName);
```

Comment:
for consistency either always use getTableName() or use getTableSchema->tableName

Reply:
Changed to `getTableName()`